test/functionalTest/cases/2193_v1_reply_to_v2_response_on_error_with_cpr/forward_using_GET_v2_entities.test

```diff
@@ -0,0 +1,220 @@
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
Actually, sufix vs. prefix is probably a matter of taste (at least I don't remember any style rule about it). Thus, sufixes are also ok if you prefer so.